src/native/managed/cdacreader/src/Legacy/SOSDacImpl.cs

```diff
@@ -349,16 +349,22 @@ int ISOSDacInterface.GetMethodDescData(ulong methodDesc, ulong ip, DacpMethodDes
                 }
             }
 
-#if false // TODO[cdac]: HAVE_GCCOVER
+            // HAVE_GCCOVER
             if (requestedNativeCodeVersion.Valid)
             {
-                TargetPointer gcCoverAddr = nativeCodeContract.GetGCCoverageInfo(requestedNativeCodeVersion);
-                if (gcCoverAddr != TargetPointer.Null)
+                TargetPointer? gcCoverAddr = _target.Contracts.GCCover.GetGCCoverageInfo(requestedNativeCodeVersion);
+                if (gcCoverAddr is not null)
                 {
-                    throw new NotImplementedException(); // TODO[cdac]: gc stress code copy
+                    // HAVE_GCCOVER is enabled
+                    if (gcCoverAddr.Value != TargetPointer.Null)
```

Comment:
What case would `gcCoverAddr` be non-null, but have a `TargetPointer.Null` value? This is precisely the issue with using null as a sentinel concept, it is ambiguous here.

Reply:
`gcCoverAddr` would be a `null` value if we can't fetch the field from the target, when `HAVE_GCCOVER` is disabled. `gcCoverAddr` could contain a `TargetPointer.Null` when `HAVE_GCCOVER` is enabled but the field in the target is a `nullptr`. 

This distinction isn't important for the current `GetMethodDescData` implementation, but I thought it was worth saving in case we want to know if `HAVE_GCCOVER` is enabled in the target.

Reply:
Based on comments above, this information is not required, and I simplified the null check.